packages/host/src/AppBar.tsx

```diff
@@ -1,108 +1,43 @@
-import React, { useState } from 'react';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
 import {
   ArrowLeft,
   Book,
   Button,
   styled,
   Toolbar,
-  Typography,
   useDrawer,
   useHostContext,
-  useTranslation,
   Box,
   UnstyledIconButton,
+  Breadcrumbs,
 } from '@openmsupply-client/common';
 import { LanguageMenu } from './LanguageMenu';
-import { useNavigate, Link, useLocation } from 'react-router-dom';
-import { LocaleKey } from '@openmsupply-client/common/src/intl/intlHelpers';
-
-const Breadcrumb = styled(Link)({
-  color: 'inherit',
-  fontWeight: 'bold',
-  textDecoration: 'none',
-});
-
-const ArrowIcon = styled(ArrowLeft)({
-  marginRight: 8,
-});
-
-const H6 = styled(Typography)({
-  flexGrow: 0,
-});
+import { ExternalURL } from '@openmsupply-client/config';
 
 const ButtonContainer = styled('div')({
   display: 'flex',
   flex: 1,
   justifyContent: 'flex-end',
 });
 
-const StyledToolbar = styled(Toolbar)({ paddingRight: 0 });
-
-interface urlPart {
-  path: string;
-  key: LocaleKey;
-  value: string;
-}
-
-const Breadcrumbs: React.FC = () => {
-  const t = useTranslation();
-  const location = useLocation();
-  const [urlParts, setUrlParts] = useState<urlPart[]>([]);
-
-  React.useEffect(() => {
-    const parts = location.pathname.split('/');
-    const urlParts: urlPart[] = [];
-
-    parts.reduce((fullPath, part) => {
-      if (part === '') return '';
-      const path = `/${fullPath}/${part}`;
-      urlParts.push({ path, key: `app.${part}` as LocaleKey, value: part });
-      return path;
-    }, '');
-    setUrlParts(urlParts);
-  }, [location]);
-
-  const crumbs = urlParts.map((part, index) => {
-    if (index === urlParts.length - 1) {
-      const title = /^\d+$/.test(part.value)
-        ? t('breadcrumb.item', { id: part.value })
-        : t(part.key);
-      return <span key={part.key}>{title}</span>;
-    }
-
-    return (
-      <span key={part.key}>
-        <Breadcrumb to={part.path}>{t(part.key)}</Breadcrumb>
-        {' / '}
-      </span>
-    );
-  });
-
-  return (
-    <H6 variant="h6" color="inherit" noWrap>
-      {crumbs}
-    </H6>
-  );
-};
-
 const StyledContainer = styled(Box, {
   shouldForwardProp: prop => prop !== 'isOpen',
 })<{ isOpen: boolean }>(({ isOpen, theme }) => ({
-  left: 72,
-  position: 'absolute',
+  marginLeft: 80,
+  marginRight: 0,
+  minHeight: 90,
+
+  zIndex: theme.zIndex.drawer - 1,
+  boxShadow: theme.shadows[1],
+
```

Comment:
I changed this to not use `position` absolute (and am hoping thats OK) and just did some adjustments to the width/margin/left offset and it seemed to be ok..

I saw mui had an `AppBar` component and tried figuring out if we should be using that but css is so confusing 😢

Reply:
I actually think better to not use absolute if it can be avoided. as far as I can tell, we don't need it - the page length is constrained, so we don't have a need to stick the app bar to the top.

I had a look at the Mui AppBar to see if it gave us anything and from what I could see, it only gave us pain. So I stuck with this approximation instead.